src/app/services/xml-parser-service/xmp-parser.service.ts

```diff
@@ -0,0 +1,179 @@
+import {
+  org,
+  ManifestParser as manifestParser
+} from '@cruglobal/godtools-shared';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+export type Text = org.cru.godtools.shared.tool.parser.model.Text;
+export type Content = org.cru.godtools.shared.tool.parser.model.Content;
+export type Image = org.cru.godtools.shared.tool.parser.model.Image;
+export type Page = org.cru.godtools.shared.tool.parser.model.page.Page;
+export type CallToAction =
+  org.cru.godtools.shared.tool.parser.model.tract.CallToAction;
+export type Header = org.cru.godtools.shared.tool.parser.model.tract.Header;
+export type Hero = org.cru.godtools.shared.tool.parser.model.tract.Hero;
+export type Modal = org.cru.godtools.shared.tool.parser.model.tract.Modal;
+export type TractPage =
+  org.cru.godtools.shared.tool.parser.model.tract.TractPage;
+export type Card =
+  org.cru.godtools.shared.tool.parser.model.tract.TractPage.Card;
+export type Parent = org.cru.godtools.shared.tool.parser.model.Parent;
+export type Paragraph = org.cru.godtools.shared.tool.parser.model.Paragraph;
+export type Resource = org.cru.godtools.shared.tool.parser.model.Resource;
+export type Manifest = org.cru.godtools.shared.tool.parser.model.Manifest;
+export type XmlParserData =
+  org.cru.godtools.shared.tool.parser.ParserResult.Data;
+export type Video = org.cru.godtools.shared.tool.parser.model.Video;
+export type Button = org.cru.godtools.shared.tool.parser.model.Button;
+export type EventId = org.cru.godtools.shared.tool.parser.model.EventId;
+export type Form = org.cru.godtools.shared.tool.parser.model.Form;
+export type Input = org.cru.godtools.shared.tool.parser.model.Input;
+export type Link = org.cru.godtools.shared.tool.parser.model.Link;
+
+export type Spacer = org.cru.godtools.shared.tool.parser.model.Spacer;
+export type Tabs = org.cru.godtools.shared.tool.parser.model.Tabs;
+export type Tab = org.cru.godtools.shared.tool.parser.model.Tabs.Tab;
+export type Accordion = org.cru.godtools.shared.tool.parser.model.Accordion;
+export type ModalCard = org.cru.godtools.shared.tool.parser.model.Card;
```

Comment:
connecting to my previous comment, this probably should be what you export as `Card` from here, since it's a standard `Card` that can be used anywhere